File Picker UX

odeimaiz opened the pull request:
## What do these changes do?
- [x] Progress to 100% when file selected
- [x] Select file when reopening FilePicker if any preselected

## Related issue number
closes #806

![FP](https://user-images.githubusercontent.com/33152403/58263496-62e37000-7d7c-11e9-8f3b-0a0523474229.gif)

## Checklist
- [ ] I think the code is well written
- [ ] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [ ] **Runs in the swarm**
- [ ] If you design a new module, add your user to .github/CODEOWNERS

oetiker commented on services/web/client/source/class/qxapp/file/FilePicker.js:
```diff
@@ -126,17 +127,35 @@ qx.Class.define("qxapp.file.FilePicker", {
+    },
+
+    __modelChanged: function() {
+      console.log("modelChanged");
```
I would not add console output in PR code

sanderegg commented on services/web/client/source/class/qxapp/data/model/Workbench.js:
```diff
@@ -300,7 +300,7 @@ qx.Class.define("qxapp.data.model.Workbench", {
       const nodes = Object.values(allNodes);
       for (const node of nodes) {
-        if (!node.isDynamic()) {
+        if (node.isComputational() && !node.isInKey("file-picker")) {
```
why is the file picker a computational service? it's pretty much dynamic isn't it?

odeimaiz commented:
@ignapas 

* Select button is always disabled
Fixed

* Uploaded file does not select
Implemented

* Progress bar shows no progress
* File gets uploaded but it doesn't show until you go out and back to the file picker
Unfortunately, we can't do file related UX testing since storage service doesn't properly work in local environment, that's why there are issues with the progress and tree-rebuilding after upload.

Regarding the aesthetic changes, I completely agree. Feel free to beautify it!
